test/tracing-messaging-with-outbox.test.js

```diff
@@ -8,9 +8,7 @@ const env = {
 process.env.cds_requires_messaging = JSON.stringify(env)
 
 const CHECK = (log, expect) => {
-  // 2: no outbox -> consumer gets new root context
-  // REVISIT: for some reason, the emit done in the on succeeded callback gets a new root context when running in jest
-  expect(log.output.match(/\[telemetry\] - elapsed times:/g).length).to.equal(3) //> actually 2
+  expect(log.output.match(/\[telemetry\] - elapsed times:/g).length).to.equal(4)
```

Comment:
`log.output` in cds 8:
![image](https://github.com/user-attachments/assets/c7f6cbf5-3568-401f-820d-94fbcbb35c17)

`log.output`in cds 9:
![image](https://github.com/user-attachments/assets/963c79ae-5b47-4060-8e34-02448fca78a9)

Reply:
i think we should deactivate the queue for the existing tests and then add extra ones for it

Reply:
right, but that worked because the persistent outbox was off by default, i.e., the disabling was omitted. now, we need to explicitly set the in memory outbox: https://github.com/cap-js/telemetry/pull/338/files#diff-08b958738e72f5b16d4eb669fe005de06cf9dfc430ecda1494064d1dec87c93bR5